email-connector-services/src/main/java/org/exoplatform/emailConnector/listener/AnalyticsEmailListener.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * This file is part of the Meeds project (https://meeds.io/).
```

Comment:
This is the header for meeds code.

Is it code ? or should it be the header for exo ?